src/components/RequirementView.vue

```diff
@@ -74,43 +76,37 @@ import RequirementHeader from '@/components/RequirementHeader.vue';
 // eslint-disable-next-line import/extensions
 import SubRequirement from '@/components/SubRequirement.vue';
 
-import { AppUser } from '@/user-data';
+import { SingleMenuRequirement } from '@/requirements/types';
+import { AppUser, AppMajor, AppMinor } from '@/user-data';
 
 Vue.component('requirementheader', RequirementHeader);
 Vue.component('subrequirement', SubRequirement);
 
-export type Major = {
-  display: boolean;
-  readonly major: string;
-  readonly majorFN: string;
-}
-export type Minor = {
-  display: boolean;
-  readonly minor: string;
-  readonly minorFN: string;
-}
+// reqGroupColorMap maps reqGroup to an array [<hex color for progress bar>, <color for arrow image>]
+const reqGroupColorMap = {
+  UNIVERSITY: ['508197', 'grayblue'],
```

Comment:
Could we remove UNIVERSITY from the reqGroupColorMap since we won't show the UNIVERSITY group separately?

Reply:
removed it